packages/core/agent/src/plugins/epoch-monitor/epoch-monitor.ts

```diff
@@ -110,4 +74,49 @@ export class EpochMonitor extends AbstractPlugin {
     this._spaceStates.forEach((state) => state.subscriptions.forEach((subscription) => subscription.unsubscribe()));
     this._spaceStates.clear();
   }
+
+  // TODO(burdon): Subscribe to space members to update address book.
+  private async _monitorSpace(space: Space, state: SpaceState) {
+    state.subscriptions.push(
```

Comment:
should this be extracted to a separate plugin?

Reply:
How so? epoch-monitor is the plugin right?